docs/service-introspection-onboarding.md

```diff
@@ -25,101 +48,134 @@ following properties of this storage before proceeding:
-   every build that runs from the source repository to show up in Spektate.
-
-   ```yaml
-   - bash: |
-       curl $SCRIPT > script.sh
-       chmod +x ./script.sh
-       tag_name="hello-spektate-$(Build.SourceBranchName)-$(Build.BuildId)"
-       commitId=$(Build.SourceVersion)
-       commitId=$(echo "${commitId:0:7}")
-       ./script.sh $(ACCOUNT_NAME) $(ACCOUNT_KEY) $(TABLE_NAME) $(PARTITION_KEY) p1 $(Build.BuildId) imageTag $tag_name commitId $commitId service $(Build.Repository.Name)
-     displayName: Update manifest pipeline details in CJ db
-     env:
-       SCRIPT: https://raw.githubusercontent.com/catalystcode/spk/master/scripts/update_introspection.sh
-   ```
-
-   Make sure the variable `tag_name` is set to the tag name for the image being
-   built in your docker step.
-
-   **Note**: The earlier in the pipeline you add this task, the earlier it will
-   send data to Spektate. Adding it before the crucial steps is recommended
-   since it will capture details about failures if the important steps fail.
-
-3. To your CD release pipeline (ACR to HLD), add the following lines of code to
-   the end of your last release task (make sure this is not a separate task in
-   the process):
-
-   ```yaml
+**Note:** The Azure storage account is needed to store information about your
+pipelines and services that is displayed by service introspection.
+
+2. Create a table. Follow these
+   [instructions](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+
+### Pipelines Configuration
+
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. This is done by adding a script snippet in each
+`azure-pipelines.yml` configuration.
+
+#### 1. Configure a variable group
+
+To send data from Azure pipelines to the service introspection storage created
+previously a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+To configure the variable group run:
+
+```
+spk variable-group create
+```
+
+#### 2. CI pipeline configuration
+
+The CI pipeline runs from the source repository to build a docker image.
+
+Paste the following task in its corresponding `azure-pipelines.yml`:
+
+```yaml
+- bash: |
+    curl $SCRIPT > script.sh
+    chmod +x ./script.sh
+    tag_name="hello-spektate-$(Build.SourceBranchName)-$(Build.BuildId)"
+    commitId=$(Build.SourceVersion)
+    commitId=$(echo "${commitId:0:7}")
+    ./script.sh $(ACCOUNT_NAME) $(ACCOUNT_KEY) $(TABLE_NAME) $(PARTITION_KEY) p1 $(Build.BuildId) imageTag $tag_name commitId $commitId service $(Build.Repository.Name)
+  displayName: Update manifest pipeline details in CJ db
+  env:
+    SCRIPT: https://raw.githubusercontent.com/catalystcode/spk/master/scripts/update_introspection.sh
+```
+
+This task will update the service introspection storage table for every build
+that runs from the source repository. This information will be available for use
+by service introspection.
```

Comment:
Just a heads up that with the changes coming in from PR https://github.com/CatalystCode/spk/pull/95, this bash task will change a lot.